Makefile

```diff
@@ -54,6 +54,7 @@ conda-lock.yml: pyproject.toml
 .PHONY: pudl-dev
 pudl-dev:
 	${mamba} run --name base ${mamba} env remove --name pudl-dev
+	${mamba} run --name base ${mamba} install --quiet --yes conda-lock
```

Comment:
This is very minor, but I would probably do this as a first step. In case this somehow fails, you won't end up with your previous env nuked.